From 9041ce786f5ba513613db33ce810b17f7c3b0f75 Mon Sep 17 00:00:00 2001
From: wouterlucas <wouter@wouterlucas.com>
Date: Thu, 26 Jul 2018 22:27:03 -0700
Subject: [PATCH] Initial gstreamer-wpe support

---
 common/m4/gst-check.m4 | 32 ++++++++++++++++----------------
 configure.ac           |  2 +-
 2 files changed, 17 insertions(+), 17 deletions(-)

diff --git a/common/m4/gst-check.m4 b/common/m4/gst-check.m4
index 4277053..de227dd 100644
--- a/common/m4/gst-check.m4
+++ b/common/m4/gst-check.m4
@@ -66,10 +66,10 @@ AC_DEFUN([AG_GST_CHECK_MODULES],
 
 AC_DEFUN([AG_GST_CHECK_GST],
 [
-  AG_GST_CHECK_MODULES(GST, gstreamer-[$1], [$2], [GStreamer], [$3])
+  AG_GST_CHECK_MODULES(GST, wpe-gstreamer-[$1], [$2], [GStreamer], [$3])
   dnl allow setting before calling this macro to override
   if test -z $GST_TOOLS_DIR; then
-    GST_TOOLS_DIR=`$PKG_CONFIG --variable=toolsdir gstreamer-[$1]`
+    GST_TOOLS_DIR=`$PKG_CONFIG --variable=toolsdir wpe-gstreamer-[$1]`
     if test -z $GST_TOOLS_DIR; then
       AC_MSG_ERROR(
         [no tools dir set in GStreamer pkg-config file, core upgrade needed.])
@@ -82,7 +82,7 @@ AC_DEFUN([AG_GST_CHECK_GST],
   dnl this is used for unit tests
   dnl allow setting before calling this macro to override
   if test -z $GST_PLUGINS_DIR; then
-    GST_PLUGINS_DIR=`$PKG_CONFIG --variable=pluginsdir gstreamer-[$1]`
+    GST_PLUGINS_DIR=`$PKG_CONFIG --variable=pluginsdir wpe-gstreamer-[$1]`
     if test -z $GST_PLUGINS_DIR; then
       AC_MSG_ERROR(
         [no pluginsdir set in GStreamer pkg-config file, core upgrade needed.])
@@ -94,25 +94,25 @@ AC_DEFUN([AG_GST_CHECK_GST],
 
 AC_DEFUN([AG_GST_CHECK_GST_BASE],
 [
-  AG_GST_CHECK_MODULES(GST_BASE, gstreamer-base-[$1], [$2],
+  AG_GST_CHECK_MODULES(GST_BASE, wpe-gstreamer-base-[$1], [$2],
     [GStreamer Base Libraries], [$3])
 ])
 
 AC_DEFUN([AG_GST_CHECK_GST_CONTROLLER],
 [
-  AG_GST_CHECK_MODULES(GST_CONTROLLER, gstreamer-controller-[$1], [$2],
+  AG_GST_CHECK_MODULES(GST_CONTROLLER, wpe-gstreamer-controller-[$1], [$2],
     [GStreamer Controller Library], [$3])
 ])
 
 AC_DEFUN([AG_GST_CHECK_GST_NET],
 [
-  AG_GST_CHECK_MODULES(GST_NET, gstreamer-net-[$1], [$2],
+  AG_GST_CHECK_MODULES(GST_NET, wpe-gstreamer-net-[$1], [$2],
     [GStreamer Network Library], [$3])
 ])
 
 AC_DEFUN([AG_GST_CHECK_GST_CHECK],
 [
-  AG_GST_CHECK_MODULES(GST_CHECK, gstreamer-check-[$1], [$2],
+  AG_GST_CHECK_MODULES(GST_CHECK, wpe-gstreamer-check-[$1], [$2],
     [GStreamer Check unittest Library], [$3])
 ])
 
@@ -160,7 +160,7 @@ dnl GST_PLUGINS_DIR anyway.
 dnl ===========================================================================
 AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_BASE],
 [
-  AG_GST_CHECK_MODULES(GST_PLUGINS_BASE, gstreamer-plugins-base-[$1], [$2],
+  AG_GST_CHECK_MODULES(GST_PLUGINS_BASE, wpe-gstreamer-plugins-base-[$1], [$2],
     [GStreamer Base Plugins], [$3])
 
   if test "x$HAVE_GST_PLUGINS_BASE" = "xyes"; then
@@ -168,7 +168,7 @@ AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_BASE],
     dnl this is used for unit tests
     dnl allow setting before calling this macro to override
     if test -z $GSTPB_PLUGINS_DIR; then
-      GSTPB_PLUGINS_DIR=`$PKG_CONFIG --variable=pluginsdir gstreamer-plugins-base-[$1]`
+      GSTPB_PLUGINS_DIR=`$PKG_CONFIG --variable=pluginsdir wpe-gstreamer-plugins-base-[$1]`
       if test -z $GSTPB_PLUGINS_DIR; then
         AC_MSG_ERROR(
           [no pluginsdir set in GStreamer Base Plugins pkg-config file])
@@ -190,7 +190,7 @@ dnl an installed setup all plugins will be found in GST_PLUGINS_DIR anyway.
 dnl ===========================================================================
 AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_GOOD],
 [
-  AG_GST_CHECK_MODULES(GST_PLUGINS_GOOD, gstreamer-plugins-good-[$1], [$2],
+  AG_GST_CHECK_MODULES(GST_PLUGINS_GOOD, wpe-gstreamer-plugins-good-[$1], [$2],
     [GStreamer Good Plugins], [no])
 
   if test "x$HAVE_GST_PLUGINS_GOOD" = "xyes"; then
@@ -198,7 +198,7 @@ AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_GOOD],
     dnl this is used for unit tests
     dnl allow setting before calling this macro to override
     if test -z $GST_PLUGINS_GOOD_DIR; then
-      GST_PLUGINS_GOOD_DIR=`$PKG_CONFIG --variable=pluginsdir gstreamer-plugins-good-[$1]`
+      GST_PLUGINS_GOOD_DIR=`$PKG_CONFIG --variable=pluginsdir wpe-gstreamer-plugins-good-[$1]`
       if test -z $GST_PLUGINS_GOOD_DIR; then
         AC_MSG_ERROR([no pluginsdir set in GStreamer Good Plugins pkg-config file])
       fi
@@ -218,7 +218,7 @@ dnl an installed setup all plugins will be found in GST_PLUGINS_DIR anyway.
 dnl ===========================================================================
 AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_UGLY],
 [
-  AG_GST_CHECK_MODULES(GST_PLUGINS_UGLY, gstreamer-plugins-ugly-[$1], [$2],
+  AG_GST_CHECK_MODULES(GST_PLUGINS_UGLY, wpe-gstreamer-plugins-ugly-[$1], [$2],
     [GStreamer Ugly Plugins], [no])
 
   if test "x$HAVE_GST_PLUGINS_UGLY" = "xyes"; then
@@ -226,7 +226,7 @@ AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_UGLY],
     dnl this is used for unit tests
     dnl allow setting before calling this macro to override
     if test -z $GST_PLUGINS_UGLY_DIR; then
-      GST_PLUGINS_UGLY_DIR=`$PKG_CONFIG --variable=pluginsdir gstreamer-plugins-ugly-[$1]`
+      GST_PLUGINS_UGLY_DIR=`$PKG_CONFIG --variable=pluginsdir wpe-gstreamer-plugins-ugly-[$1]`
       if test -z $GST_PLUGINS_UGLY_DIR; then
         AC_MSG_ERROR([no pluginsdir set in GStreamer Ugly Plugins pkg-config file])
       fi
@@ -246,7 +246,7 @@ dnl an installed setup all plugins will be found in GST_PLUGINS_DIR anyway.
 dnl ===========================================================================
 AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_BAD],
 [
-  AG_GST_CHECK_MODULES(GST_PLUGINS_BAD, gstreamer-plugins-bad-[$1], [$2],
+  AG_GST_CHECK_MODULES(GST_PLUGINS_BAD, wpe-gstreamer-plugins-bad-[$1], [$2],
     [GStreamer Bad Plugins], [no])
 
   if test "x$HAVE_GST_PLUGINS_BAD" = "xyes"; then
@@ -254,7 +254,7 @@ AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_BAD],
     dnl this is used for unit tests
     dnl allow setting before calling this macro to override
     if test -z $GST_PLUGINS_BAD_DIR; then
-      GST_PLUGINS_BAD_DIR=`$PKG_CONFIG --variable=pluginsdir gstreamer-plugins-bad-[$1]`
+      GST_PLUGINS_BAD_DIR=`$PKG_CONFIG --variable=pluginsdir wpe-gstreamer-plugins-bad-[$1]`
       if test -z $GST_PLUGINS_BAD_DIR; then
         AC_MSG_ERROR([no pluginsdir set in GStreamer Bad Plugins pkg-config file])
       fi
@@ -282,7 +282,7 @@ AC_DEFUN([AG_GST_CHECK_GST_PLUGINS_LIBAV],
     dnl this is used for unit tests
     dnl allow setting before calling this macro to override
     if test -z $GST_PLUGINS_LIBAV_DIR; then
-      GST_PLUGINS_LIBAV_DIR=`$PKG_CONFIG --variable=pluginsdir gstreamer-plugins-libav-[$1]`
+      GST_PLUGINS_LIBAV_DIR=`$PKG_CONFIG --variable=pluginsdir wpe-gstreamer-plugins-libav-[$1]`
       if test -z $GST_PLUGINS_LIBAV_DIR; then
         AC_MSG_ERROR([no pluginsdir set in GStreamer Libav Plugins pkg-config file])
       fi
diff --git a/configure.ac b/configure.ac
index b1ef5fa..0f525ea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -158,7 +158,7 @@ AM_CONDITIONAL(HAVE_GST_GL, test "x$GST_GL" = "xyes")
 
 dnl Check for documentation xrefs
 GLIB_PREFIX="`$PKG_CONFIG --variable=prefix glib-2.0`"
-GST_PREFIX="`$PKG_CONFIG --variable=prefix gstreamer-$GST_API_VERSION`"
+GST_PREFIX="`$PKG_CONFIG --variable=prefix gstreamer-wpe-$GST_API_VERSION`"
 AC_SUBST(GLIB_PREFIX)
 AC_SUBST(GST_PREFIX)
 
-- 
2.17.0

