From 08d597bb87c736c52b611762e71a119dddc9a759 Mon Sep 17 00:00:00 2001
From: Michal Bugno <michalbugno1@gmail.com>
Date: Tue, 10 Aug 2021 01:12:10 -0700
Subject: [PATCH] Using wpe gstreamer instead of generic one

---
 dpi/gstreamer/CMakeLists.txt | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/dpi/gstreamer/CMakeLists.txt b/dpi/gstreamer/CMakeLists.txt
index 2b0f9c4..ca6c030 100755
--- a/dpi/gstreamer/CMakeLists.txt
+++ b/dpi/gstreamer/CMakeLists.txt
@@ -26,11 +26,11 @@ find_package(Freetype REQUIRED)
 
 find_package(PkgConfig)
 
-pkg_check_modules(GST REQUIRED gstreamer-1.0>=1.4
-                               gstreamer-sdp-1.0>=1.4
-                               gstreamer-video-1.0>=1.4
-                               gstreamer-app-1.0>=1.4
-                               gstreamer-audio-1.0>=1.4) 
+pkg_check_modules(GST REQUIRED wpe-gstreamer-1.0>=1.4
+                               wpe-gstreamer-sdp-1.0>=1.4
+                               wpe-gstreamer-video-1.0>=1.4
+                               wpe-gstreamer-app-1.0>=1.4
+                               wpe-gstreamer-audio-1.0>=1.4) 
 
 
 find_package(WPEFramework REQUIRED)
-- 
2.7.4

