From 6c468f780d8b9808fa1ca36c15abc7e3d4cdb0c2 Mon Sep 17 00:00:00 2001
From: Michal Bugno <michalbugno1@gmail.com>
Date: Fri, 6 Aug 2021 01:31:15 -0700
Subject: [PATCH] Used wpe-gstreamer instead of generic one

---
 dpi/gstreamer/CMakeLists.txt | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/dpi/gstreamer/CMakeLists.txt b/dpi/gstreamer/CMakeLists.txt
index fc3d140..4bb85a1 100755
--- a/dpi/gstreamer/CMakeLists.txt
+++ b/dpi/gstreamer/CMakeLists.txt
@@ -25,11 +25,11 @@ find_package(JPEG REQUIRED)
 find_package(Freetype REQUIRED)
 
 find_package(PkgConfig)
-pkg_check_modules(GST REQUIRED gstreamer-1.0>=1.4
-                               gstreamer-sdp-1.0>=1.4
-                               gstreamer-video-1.0>=1.4
-                               gstreamer-app-1.0>=1.4
-                               gstreamer-audio-1.0>=1.4) 
+pkg_check_modules(GST REQUIRED wpe-gstreamer-1.0>=1.4
+                               wpe-gstreamer-sdp-1.0>=1.4
+                               wpe-gstreamer-video-1.0>=1.4
+                               wpe-gstreamer-app-1.0>=1.4
+                               wpe-gstreamer-audio-1.0>=1.4) 
 
 
 find_package(WPEFramework REQUIRED)
-- 
2.7.4

