From aefb10ef2fdbb4227ff21d1fbc862aa87423a9da Mon Sep 17 00:00:00 2001
From: wouterlucas <wouter@wouterlucas.com>
Date: Thu, 26 Jul 2018 22:34:54 -0700
Subject: [PATCH] Initial support for wpe-gstreamer

---
 configure.ac | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8da9c73..6b43e7b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -230,14 +230,14 @@ AC_ARG_ENABLE(gstreamer1,
 
 
 AC_MSG_NOTICE([Look for Gstreamer-1.4])
-PKG_CHECK_MODULES([GST], [gstreamer-1.0 >= 1.4], have_gst1="yes", have_gst1="no")
+PKG_CHECK_MODULES([GST], [wpe-gstreamer-1.0 >= 1.4], have_gst1="yes", have_gst1="no")
 AS_IF([test "x$have_gst1" = "xyes"], [
     GST_MAJORMINOR=1.0
-    PKG_CHECK_MODULES([GSTBASE], [gstreamer-base-1.0 >= 1.4])
-    PKG_CHECK_MODULES([GSTRTP], [gstreamer-rtp-1.0 >= 1.4])
-    PKG_CHECK_MODULES([GSTAUDIO], [gstreamer-audio-1.0 >= 1.4])
-    PKG_CHECK_MODULES([GSTRIFF], [gstreamer-riff-1.0 >= 1.4])
-    PKG_CHECK_MODULES([GSTTAG], [gstreamer-tag-1.0 >= 1.4])
+    PKG_CHECK_MODULES([GSTBASE], [wpe-gstreamer-base-1.0 >= 1.4])
+    PKG_CHECK_MODULES([GSTRTP], [wpe-gstreamer-rtp-1.0 >= 1.4])
+    PKG_CHECK_MODULES([GSTAUDIO], [wpe-gstreamer-audio-1.0 >= 1.4])
+    PKG_CHECK_MODULES([GSTRIFF], [wpe-gstreamer-riff-1.0 >= 1.4])
+    PKG_CHECK_MODULES([GSTTAG], [wpe-gstreamer-tag-1.0 >= 1.4])
     AC_DEFINE(USE_GST1, 1, [Build with GStreamer 1.x])
     ], [])
 
-- 
2.17.0

